bin/python/ripple/util/Sign.py

```diff
@@ -22,9 +23,9 @@
     check <key>
         Check an existing key for validity.
 
-    sign <sequence> <validator-public> <master-secret>
+    sign <sequence> <validator-public-key> <validator-private-key> <master-secret>
         Create a new signed manifest with the given sequence
-        number, validator public key, and master secret key.
+        number, validation public key, and master public key.
```

Comment:
I took a stab at cleaning up `Sign.py`. I replaced private key with `sk`, for `secret key`, and public key with `pk`. I also always prefixed these keys with either `validator` for the ephemeral keys or `master` for the master keys. I also added python docs to two of the functions. We should start adding those docs when we change code.

This is for your consideration, and of course should be reviewed (I only checked the unit tests still ran):
https://github.com/seelabs/rippled/commit/3ae2d62754bd25d92d21dda53ae009076295fed4